src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp

```diff
@@ -62,7 +62,7 @@ size_t store_emitter_params::hash() const {
 }
 
 static int get_aux_regs_as_temp(const size_t byte_size, const bool is_fill = false) {
-    if (!one_of(byte_size, 64u, 32u, 16u))
+    if (one_of(byte_size % 16, 1u, 2u, 3u))
```

Comment:
Hi Alexandra, I think this condition is correct on non-avx512 HW. There is still possibility to use aux gpr when using mask load/store instructions on avx512 when "load_size > threshold_for_mask_emu_load".
in load emitters:
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp#L418
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp#L535
Store emitter is the similar while have separate threshold: threshold_for_mask_emu_store.
If we need precisely control number of aux gpr, should also consider mask load case:-)

Reply:
Hi @chenhu-wang !
Yeah, you're right. I missed this case. Thanks a lot for the catch!
I covered this case as well. Could you take a look one more time please?